calculator/src/main/java/edu/nextstep/camp/calculator/CalculatorActivity.kt

```diff
@@ -22,10 +23,31 @@ class CalculatorActivity : AppCompatActivity() {
             viewModel = calculatorVM
         }
         setContentView(binding.root)
+        initView()
 
+        observeExpressionHistory()
         observeIncompleteExpressionErrorEvent()
     }
 
+    private fun initView() {
+        initRecyclerView()
+    }
+
+    private fun initRecyclerView() {
+        binding.recyclerView.apply {
+            adapter = calculatorHistoryAdapter
+        }
+    }
+
+    private fun observeExpressionHistory() = lifecycleScope.launch {
+        repeatOnLifecycle(Lifecycle.State.STARTED) {
+            calculatorVM.expressionHistory
+                .collect { history ->
+                    history?.let { calculatorHistoryAdapter.submitList(it) }
```

Comment:
```suggestion
                    history?.let(calculatorHistoryAdapter::submitList)
```
input의 개수 및 타입이 동일하면, 메서드 참조를 활용할 수 있어요.

Reply:
[5a2127b](https://github.com/next-step/android-calculator-mvvm/pull/113/commits/5a2127ba1ba1990fe3b572d1d35b43eaaeeb6793)
고차함수 감사합니다~